src/Library/demos/Stacks/main.blp

```diff
@@ -0,0 +1,108 @@
+using Gtk 4.0;
+using Adw 1;
+
+Box root_box {
+  orientation: vertical;
+  halign: center;
+  valign: center;
+  Stack stack {
+
+    StackPage page1 {
+      title: _("Page 1");
+      child: Adw.StatusPage {
+        title: _("Stacks");
+        description: _("only shows one of its children at a time");
+
+        Label {
+          label: _("Documentation");
+          margin-top: 24;
+          styles ["title-2"]
+        }
+
+        Box {
+          orientation: horizontal;
+          halign: center;
+          margin-top: 24;
+
+          LinkButton {
+            label: _("Stack");
+            uri: "https://docs.gtk.org/gtk4/class.Stack.html";
+          }
+
+          LinkButton {
+            label: _("Stack Sidebar");
+            uri: "https://docs.gtk.org/gtk4/class.StackSidebar.html";
+          }
+
+          LinkButton {
+            label: _("Stack Switcher");
+            uri: "https://docs.gtk.org/gtk4/class.StackSwitcher.html";
+          }
+        }
+      };
+    }
+
+    StackPage page2 {
+      title: _("Page 2");
+      child: Adw.StatusPage {
+        Adw.PreferencesGroup {
+          styles ["boxed-list"]
+          halign: center;
+
+          Adw.ComboRow navigation_row {
+            title: _("Navigation Widget");
+            model: StringList {
+              strings [_("Switch"), _("Sidebar")]
+            };
+          }
+
+          Adw.ComboRow transition_row {
+            title: _("Transition Type");
+            model: StringList {
+              strings [
+                "None", "Cross-fade", "Slide Right", "Slide Left", "Slide Up",
+                "Slide Down", "Slide Left-Right", "Slide Up-Down", "Over Up", "Over Down",
+                "Over Left", "Over Right", "Under Up", "Under Down", "Under Left", "Under Right",
+                "Over Up-Down", "Over Down-Up" , "Over Left-Right", "Over Right-Left", "Rotate Left", "Rotate Right", "Rotate Left-Right"
```

Comment:
Let's organize these into rough groupings, one grouping per-line (e.g. `Slide-*`, `Rotate-*`). I think that will make the source a bit easier to glance through.